### PR TITLE
LPS-70828 Powered by doesn't translate into Chinese correctly

### PR DESCRIPTION
**Pull request by alee8888**
@pei-jung @drewbrokke

### CODE REVIEW

File: modules/apps/frontend-theme/frontend-theme-unstyled/src/main/resources/META-INF/resources/_unstyled/templates/portal_normal.ftl [hunk 1]
```diff
@@ -63,7 +63,11 @@
 
 	<footer id="footer" role="contentinfo">
 		<p class="powered-by">
-			<@liferay.language key="powered-by" /> <a href="http://www.liferay.com" rel="external">Liferay</a>
+			<#assign powered_by_liferay =  />
```
Comment (drewbrokke) [inline modules/apps/frontend-theme/frontend-theme-unstyled/src/main/resources/META-INF/resources/_unstyled/templates/portal_normal.ftl:66]: This seems like a leftover statement

**Comment by alee8888** [comment]
ci:stop

**Comment by alee8888** [comment]
ci:test:sf

**Comment by alee8888** [comment]
ci:test:relevant

**Comment by pei-jung** [comment]
ci:test:relevant

**Comment by drewbrokke** [comment]
ci:forward

**Pull request closed without merging**